lib/pf/condition/exists_in.pm

```diff
@@ -0,0 +1,74 @@
+package pf::condition::exists_in;
+
+=head1 NAME
+
+pf::condition::exists_in - is the value exists in a hash
```

Comment:
Not sure this is grammatically correct.
Shouldn't it be : 'checks if a value exists in a hash'